Formula/pymol.rb

```diff
@@ -0,0 +1,108 @@
+class Pymol < Formula
+  include Language::Python::Virtualenv
+  desc "Molecular visualization system"
+  homepage "https://pymol.org/"
+  url "https://github.com/schrodinger/pymol-open-source/archive/v2.5.0.tar.gz"
+  sha256 "aa828bf5719bd9a14510118a93182a6e0cadc03a574ba1e327e1e9780a0e80b3"
+  license :cannot_represent
+  head "https://github.com/schrodinger/pymol-open-source.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "sip" => :build
+  depends_on "freetype"
+  depends_on "glew"
+  depends_on "glm"
+  depends_on "libpng"
+  depends_on "msgpack-cxx"
+  depends_on "netcdf"
+  depends_on "numpy"
+  depends_on "pyqt@5"
+  depends_on "python@3.10"
+  uses_from_macos "libxml2"
+
+  on_macos do
+    depends_on "libomp"
+  end
+
+  on_linux do
+    depends_on "freeglut"
+  end
+
+  resource "mmtf-cpp" do
+    url "https://github.com/rcsb/mmtf-cpp/archive/refs/tags/v1.0.0.tar.gz"
+    sha256 "881f69c4bb56605fa63fd5ca50842facc4947f686cbf678ad04930674d714f40"
+  end
```

Comment:
Shouldn't this be it's own formula?

Reply:
It could be, but it's a header-only library that's used by no one else, and doesn't meet the notability requirements for a formula anyway.